dashboard/frontend/src/services.js

```diff
@@ -1,5 +1,6 @@
-//let host = "http://localhost:8080";
-let host = "";
+import { getHost } from "./utils";
+
+const host = getHost();
```

Comment:
this gets used anywhere?

Reply:
Yes, see line 6 of the same file for example.

Reply:
ok sorry missed the deleted empty string line.